build/LibraryVersions.g.cs

```diff
@@ -34,7 +34,7 @@ public static partial class LibraryVersion
             new List<PackageBuildInfo>
             {
                 new("6.0.35"),
-                new("8.0.10", supportedFrameworks: new string[] {"net8.0"}),
+                new("8.0.10"),
```

Comment:
I don't understand why lines 37, 46, and 47 were updated to remove the `supportedFrameworks` parameter but lines 123, 124, and 174 were updated to keep the `supportedFrameworks` and a new `net9.0` TFM. Why is that?

Reply:
Lines 123, 124, and 174: NServiceBus, and WCFClient are supported both on .NET Framework and .NET.
Some of the newer version of tested libraries are working only on .NET (currently 8 and 9) so I need to add target frameworks.

37, 46, and 47: EntityFrameworkCore and EntityFrameworkCore.Pomelo.MySql are supported only on .NET. As we dropped support for .NET6 and .NET7 there is no longer to explicitly mark that 8.* versions are supported only on .NET8+.  [Version 9.0 of EFC](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore/9.0.0#supportedframeworks-body-tab) is also supporting .NET8+, so there will be changes in the coverage in near future. I have a plant update packages as a separate step.

Reply:
Ah understood. I had not thought that the changes would be package-specific. Thanks for the follow-up!